core/js/src/files/PathsJs.kt

```diff
@@ -92,6 +92,20 @@ internal class FileSource(private val path: Path) : RawSource {
     private var buffer: dynamic = null
     private var closed = false
     private var offset = 0
+    private val fd = open(path)
+
+    private fun open(path: Path): Int {
+        if (!(fs.existsSync(path.path) as Boolean)) {
+            throw FileNotFoundException("File does not exist: $path")
+        }
+        val fd = try {
+            fs.openSync(path.path, "r") as Int
```

Comment:
Generally when I know the specific numeric type, I prefer to use an `unsafeCast`.  
Casting with `as` will generate additional code paths and deeper call stack for pretty much no reasons.

Reply:
Thanks for pointing to this. I don't actually care about the exact type here, so it's fine to leave it as `dynamic`.